src/accessors/IModify.ts

```diff
@@ -0,0 +1,241 @@
+import { IMessage, IMessageAttachment } from '../messages';
+import { IRoom, RoomType } from '../rooms';
+import { IUser } from '../users';
+
+export interface IModify {
+    update(): IModifyUpdate;
+
+    extend(): IModifyExtend;
```

Comment:
Yes, that's why I moved items out in the new builder interface version I committed yesterday.